x-pack/plugins/observability/common/utils/formatters/duration.ts

```diff
@@ -148,7 +148,7 @@ function getDurationUnitKey(max: number): DurationTimeUnit {
   if (max > toMicroseconds(10, 'minutes')) {
     return 'minutes';
   }
-  if (max > toMicroseconds(10, 'seconds')) {
```

Comment:
@dgieselaar Any idea how to check places that use this logic to ensure this change is fine?

I changed it to fix [this issue](https://github.com/elastic/kibana/issues/126895), but I am not sure, besides the reason that was updated in the test if there is any other place that needs to be checked.

Reply:
@sqren do you have a good understanding of this? it looks fine to me but want to make sure you're also on board. Though we should perhaps apply this consistently, e.g. 1ms should not be converted to 1000μs